Source/ThirdParty/embree/common/sys/vector.h

```diff
@@ -195,7 +195,7 @@ namespace embree
         items = alloc.allocate(new_alloced);
         for (size_t i=0; i<size_active; i++) {
           ::new (&items[i]) T(std::move(old_items[i]));
-          alloc.destroy(&old_items[i]);
+          old_items[i].~T();
```

Comment:
This happens to be correct, because all allocators used do exactly this, but i find it weird this is needed. @eugeneko?

Reply:
Embree should push a new version that contains the new changes for C++20, but they don't seem to have pushed it yet so I adopted that patch